### PR TITLE
Add feature to delete payment methods

### PR DESCRIPTION
**Pull request by crookedneighbor**
### Summary

Closes https://github.com/braintree/braintree-web-drop-in/issues/391

Adds vault manager option to allow customers to delete their payment methods from Drop-in.

### Checklist

- [x] Added a changelog entry

### CODE REVIEW

**Comment by crookedneighbor** [comment]
demo

**Comment by igorsilva20** [comment]
Will this be merged in the near future?

**Comment by crookedneighbor** [comment]
@igorsilva20 That's the plan!

File: src/translations/en_US.js [hunk 1]
```diff
@@ -5,6 +5,15 @@ module.exports = {
   chooseAnotherWayToPay: 'Choose another way to pay',
   chooseAWayToPay: 'Choose a way to pay',
   otherWaysToPay: 'Other ways to pay',
+  edit: 'Edit',
```
Comment (crookedneighbor) [inline src/translations/en_US.js]: Would love to get these translations approved and submitted for translation.

**Comment by patriciojofre** [comment]
Hi! this feature will have an option for disable the ability to delete the current payment method? In my case, I builded a custom view for edit registered payment methods but not allowing delete the active credit card, only addtional and no active credit cards.

If drop-in will do this by default, I'll be in problems.

**Comment by darshansawe** [comment]
Could you update the online documentation to reflect this?

**Comment by crookedneighbor** [comment]
The documentation will updated when all versions of Drop-in have been updated to have this feature. We're still waiting on the Android release at the moment.

**Pull request merged**